README.md

```diff
@@ -27,6 +27,17 @@ $ impl 's *Source' golang.org/x/oauth2.TokenSource
 func (s *Source) Token() (*oauth2.Token, error) {
     panic("not implemented")
 }
+
+# You can request that the declaring file be updated in-place by passing -u
+$ impl -u 'f *File' io.ReadWriteCloser
+
+# You can also specify a position at which you'd like the generated code to be
+# inserted, helpful for editor/ide integration.
+$ impl -p main.go:12 'f *File' io.ReadWriteCloser
```

Comment:
Nit: usually the way editors take and give positions is file offset bytes. See how guru works. for example. Otherwise you get into the nastiness that is counting columns. Are they bytes? Are they runes? Are they counting visual width? All are potentially valid answers.